packages/storybook/stories/Localization.stories.mdx

```diff
@@ -8,3 +21,86 @@ Localization is a key to making products that can be used across the world and b
 UI Library will provide out of the box support for some languages and capabilities such as RTL.
 Developers can provide their own localization files to be used for the UI Library.
 These localization capabilities will be added ahead of General Availability.
+
+## LocalizationProvider
+
+`LocalizationProvider` is a wrapping component that sets all the strings for ACS components using a `Locale`. We have a set of
+locales available for import to use. LocalizationProvider is not required as all components default to using our English (US) locale.
+
+### Usage
+
+To use the LocalizationProvider, simply import it from `@azure/communication-react` and wrap it around ACS UI SDK components,
+For the example below we will localize buttons in a `ControlBar` in French (France) and import that locale `fr_FR` from `@azure/communication-react`.
+
+<Source code={LocalizationProviderSnippetText} />
+
+<Canvas withSource="none">
+  <LocalizationProviderSnippet />
+</Canvas>
+
+## Overriding our locales
+
+There are two options to customize the locales that we provide. For the examples below, let's say you wish to have a `ControlBar`
+with strings from our British English locale but you want to change the label of `CameraButton` to "Start" when its state is off.
+
+### Option 1: Import and customize locales
+
+The first option is to import and customize locales. First, import `en_GB` locale from `@azure/communication-react` and then reassign the
+value `strings.cameraButton.offLabel` like shown below.
+
+<Source code={CustomLocaleSnippetText} />
+
+<Canvas withSource="none">
+  <CustomLocaleSnippet />
+</Canvas>
+
+`Locale` stores all strings in the `strings` property. The component name is the first identifer and the string key is the second identifier such
+that each string can be accessed in this pattern `strings.<componentName>.<stringKey>`. If you are using typescript and have intellisense available
+in your IDE, it will be easy to discover all the strings.
+
+### Option 2: Component strings prop
+
+The second option is also easy to pass custom strings directly through the `strings` prop. Each component in ACS has an optional `strings`
```

Comment:
General comment: Drop words like "simple" and "easy". It can come off as being hand-wavy (or worse, condescending).